docs/guides/stm32_getting_started_guide.md

```diff
@@ -0,0 +1,103 @@
+# STM32 Getting Started Guide
+
+The stm32 platform uses
+[Matter](https://github.com/project-chip/connectedhomeip) sdk with FreeRTOS.
+
+---
+
+-   [stm32 Getting Started Guide](#stm32-getting-started-guide)
+    -   [Supported Chips](#supported-chips)
+    -   [Matter Example Applications](#matter-example-applications)
+    -   [Building the Example Application](#building-the-example-application)
+    -   [Commissioning](#commissioning)
+    -   [BLE-Thread mode](#ble-thread-mode)
+
+---
+
+## Supported Chips
+
+The stm32 platform is supported on:
+
+-   [STM32WB5MM-DK](https://www.st.com/en/evaluation-tools/stm32wb5mm-dk.html)
+
+## Matter Example Applications
+
+Sample Matter applications are provided for the stm32 platform. They can be used
+to speed up development. You can find them in the samples with `/stm32`
+subfolder.
+
+## Building the Example Application
+
+-   [Setup Matter Environment](./BUILDING.md)
+
+-   Setup STLINK tools
```

Comment:
```suggestion
-   Set up STLINK tools
```
Or whatever capitalization you want here; either "Up" and "Tools", or "up" and "tools"